pkg/hints/pow_hints.go

```diff
@@ -10,13 +10,15 @@ import (
 // Implements hint:
 // %{ ids.locs.bit = (ids.prev_locs.exp % PRIME) & 1 %}
 func pow(ids IdsManager, vm *VirtualMachine) error {
-	prev_locs_exp_addr, err := ids.GetAddr("prev_locs", vm)
-	prev_locs_exp, _ := vm.Segments.Memory.GetFelt(prev_locs_exp_addr.AddUint(4))
+	prev_locs_exp_addr, err := ids.GetRelocatable("prev_locs", vm)
+	if err != nil {
+		return err
+	}
 
+	prev_locs_exp, err := vm.Segments.Memory.GetFelt(prev_locs_exp_addr.AddUint(4))
```

Comment:
```suggestion
	prev_locs_exp_addr, err := ids.GetStructFieldFelt("prev_locs", 4, vm)
```

Reply:
I've been trying to make this work for a while, but I wasn't able to. I think there is a dereference somewhere that might not be handled.
Here is how `prev_locs` is initialized in the cairo function that calls the hint:
https://github.com/starkware-libs/cairo-lang/blob/27a157d761ae49b242026bcbe5fca6e60c1e98bd/src/starkware/cairo/common/pow.cairo#L25

Reply:
Oh sorry, my bad, I thought it said GetAddr instead of GetRelocatable